config.yaml

```diff
@@ -8,32 +8,24 @@
 ---
 Standards:
   -
-    Title: Binary Numbers
-    Description: Binary Numbers
-    UID: 7985c55db967192267ccd8c6db86d270
+    Title: About PSEs
+    Description: About Weekly Problem-Solving Exercises 
+    UID: 1e030b14-342a-49b5-89b5-4472f165cbfb
     SuccessCriteria:
       - success criteria
     ContentFiles:
       -
         Type: Lesson
         UID: a6af8fa6bab71c8b803ecaf34e0984c1
-        Path: /01-binary-numbers/01-binary-numbers.md
-      -
-        Type: Checkpoint
-        UID: 45f708d7a65ca33c0e8e298ae804f9bd
-        Path: /01-binary-numbers/02-binary-numbers-checkpoint.md
+        Path: 00-about-pses/01-about-pses.md
   -
-    Title: Arrays
-    Description: Arrays
-    UID: ff43b8de4f41d5103405ddb62eb8d34e
+    Title: PSE 1
+    Description: PSE 01
```

Comment:
Can we have a blank description? Otherwise it'll just have unnecessary text on Learn

Reply:
```suggestion
    Description: Problem Solving Exercise 1
```